deploy/scripts/src/btm/test.ts

```diff
@@ -102,21 +254,143 @@ const awsConfig = new AWSConfig({
 })
 
 const eventData = {
-  payload: __ENV.DATA_BTM_SQS_PAYLOAD
+  payload: __ENV.DATA_BTM_SQS_PAYLOAD,
+  payloadData1: __ENV.DATA_BTM_SQS_PAYLOAD_DATA1,
+  payloadData2: __ENV.DATA_BTM_SQS_PAYLOAD_DATA2,
+  payloadData3: __ENV.DATA_BTM_SQS_PAYLOAD_DATA3,
+  payloadTimestamp: __ENV.DATA_BTM_SQS_PAYLOAD_TIMESTAMP
```

Comment:
If any of these are fixed secrets add them to the Parameter Store and `deploy/template.yaml`

Reply:
Removed Payload_Data1, 2, 3, Added to SQS Payload to parameter store